cinn/optim/replace_var_with_expr.cc

```diff
@@ -135,20 +147,23 @@ struct ReplaceVarIndexOfCacheMutator : public ir::IRMutator<> {
 
  private:
   std::map<std::string, ir::Tensor>* global_tensor_map_;
+  std::unordered_set<std::string>* resized_buffer_;
```

Comment:
It's better to use `std::unordered_set<std::string>& resized_buffer_`

Reply:
Done, thanks